src/logic/Preferences.swift

```diff
@@ -197,6 +197,7 @@ enum ShowOnScreenPreference: String, CaseIterable, MacroPreference {
         switch self {
             case .active: return NSLocalizedString("Active screen", comment: "")
             case .includingMouse: return NSLocalizedString("Screen including mouse", comment: "")
+            case .first return NSLocalizedString("Screen including menu bar", comment: "")
```

Comment:
The build is failing because your forgot the `:` here:

```
/Users/travis/build/lwouis/alt-tab-macos/src/logic/Preferences.swift:200:25: error: expected ':' after 'case'
            case .first return NSLocalizedString("Screen including menu bar", comment: "")
```